devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+          bitOffset: 21
+          bitWidth: 1
+        MR22:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 22
+          bitWidth: 1
+        MR23:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 23
+          bitWidth: 1
+        MR24:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 24
+          bitWidth: 1
+        MR25:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 25
+          bitWidth: 1
+        MR26:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 26
+          bitWidth: 1
+        MR27:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 27
+          bitWidth: 1
+        MR28:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 28
+          bitWidth: 1
+        MR29:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 29
+          bitWidth: 1
+        MR30:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 30
+          bitWidth: 1
+        MR31:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 31
+          bitWidth: 1
+      C1EMR1:
+        description: EXTI event mask register
+        addressOffset: 0x00C4
+        resetValue: 0x00000000
+        fields:
+        MR0:
+          description: CPU2 event mask on Event input x
+          bitOffset: 00
+          bitWidth: 1
+        MR1:
+          description: CPU2 event mask on Event input x
+          bitOffset: 01
+          bitWidth: 1
+        MR2:
+          description: CPU2 event mask on Event input x
+          bitOffset: 02
+          bitWidth: 1
+        MR3:
+          description: CPU2 event mask on Event input x
+          bitOffset: 03
+          bitWidth: 1
+        MR4:
+          description: CPU2 event mask on Event input x
+          bitOffset: 04
+          bitWidth: 1
+        MR5:
+          description: CPU2 event mask on Event input x
+          bitOffset: 05
+          bitWidth: 1
+        MR6:
+          description: CPU2 event mask on Event input x
+          bitOffset: 06
+          bitWidth: 1
+        MR7:
+          description: CPU2 event mask on Event input x
+          bitOffset: 07
+          bitWidth: 1
+        MR8:
+          description: CPU2 event mask on Event input x
+          bitOffset: 08
+          bitWidth: 1
+        MR9:
+          description: CPU2 event mask on Event input x
+          bitOffset: 09
```

Comment:
Same comment as above re using `9` instead of `09` etc

Reply:
Tidied all leading zeros, it was an error from the regex I used to write it all this